portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/CookiesSiteLevelUseCase.testcase

```diff
@@ -0,0 +1,508 @@
+				userEmailAddress = "test@liferay.com");
+
+			CookiePreferenceHandlingAdmin.disablePreferenceHandlingWithOutCookieConsentInSiteSettings(
+				baseURL = "http://www.able.com:8080",
+				portlet = "Preference Handling",
+				siteURLKey = "guest");
+		}
+
+		task ("When: End User first visits the site") {
+			UserBar.signOut();
+
+			CookieHelper.deleteAllCookiesAndRefreshPage();
+
+			User.firstLoginUI(
+				password = "test",
+				specificURL = "http://www.able.com:8080",
+				userEmailAddress = "test@liferay.com");
+		}
+
+		task ("Then: Verify No cookies are set, only the functional cookie: GUEST_LANGUAGE_ID is set verify cookie banner is not present.") {
+			CookieBanner.verifyCookieBannerIsNotPresent();
+
+			CookieConsentPanel.verifyCookiesForCookieCategoriesAreNotPresent();
+
+			CookieHelper._checkSelectedCookieValue(
+				cookieName = "GUEST_LANGUAGE_ID",
+				expectedCookieValue = "en_US");
+		}
+	}
+
+	@description = "This is a use case for LPS-170904 TC-2. Verify only functional cookies are set in Site level and Implicit mode, when the user is a first-time visitor."
+	@priority = 5
+	test VerifyFunctionalCookiesForFirstVisitInSiteLevelImplicitMode {
+		property test.name.skip.portal.instance = "CookiesSiteLevelUseCase#VerifyFunctionalCookiesForFirstVisitInSiteLevelImplicitMode";
+
+		task ("Given: Site Administrator enabled the cookie preference handling in site level and the implicit cookie consent mode.") {
+			Site.openSiteSettingsAdmin(siteURLKey = "guest");
+
+			Site.addVirtualHostsURLCP(
+				pageVirtualHosts = "true",
+				pageVirtualHostURL = "www.able.com");
+
+			UserBar.signOut();
+
+			User.firstLoginUI(
+				password = "test",
+				specificURL = "http://www.able.com:8080",
+				userEmailAddress = "test@liferay.com");
+
+			CookiePreferenceHandlingAdmin.enablePreferenceHandlingWithOutCookieConsentInSiteSettings(
+				baseURL = "http://www.able.com:8080",
+				portlet = "Preference Handling",
+				siteURLKey = "guest");
+		}
+
+		task ("When: End User first visits the site") {
+			UserBar.signOut();
+
+			CookieHelper.deleteAllCookiesAndRefreshPage();
+
+			User.firstLoginUI(
+				password = "test",
+				specificURL = "http://www.able.com:8080",
+				userEmailAddress = "test@liferay.com");
+		}
+
+		task ("Then: Verify only the functional cookie is set cookie banner is present.") {
+			CookieBanner.verifyCookieBannerIsPresent();
+
+			CookieConsentPanel.verifyCookiesForCookieCategoriesAreNotPresent();
+
+			CookieHelper._checkSelectedCookieValue(
+				cookieName = "GUEST_LANGUAGE_ID",
+				expectedCookieValue = "en_US");
+		}
+	}
+
+	@description = "This is a use case for LPS-170904 TC-4. Previously enabled and set cookies are deleted when Site level configuration is Disabled."
+	@priority = 5
+	test VerifyFunctionalCookiesForNotFirstVisitInSiteLevelDisableImplicitMode {
+		property test.name.skip.portal.instance = "CookiesSiteLevelUseCase#VerifyFunctionalCookiesForNotFirstVisitInSiteLevelDisableImplicitMode";
```

Comment:
Please ass HttpOnly disabler for this test case

Reply:
Done